### DISCUSSION TITLE
Request to upgrade starlette version (atleast to 0.28.0)

### DISCUSSION THREAD

**Discussion by Gribesh** [fastapi/fastapi#10125] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No code :)
```

### Description

I am using Fastapi's latest version, and I am reading the request body for my middleware. Once I read the request body, I cannot reuse it later. This issue is because of starllete and it is already resolved in starlette v0.28.0. (https://www.starlette.io/release-notes/#0280)

But the fastapi use starlette v 0.27.0 , Can you please upgrade the starlette version officially?

_For now, I have resolved the dependency issue and running it._ 

### Operating System

Windows

### FastAPI Version

0.101.1

### Pydantic Version

2.2.1

### Python Version

3.10

**Comment by Kludex** [comment, marked as answer]
PR already exists: https://github.com/tiangolo/fastapi/pull/9636.

**Reply by Gribesh** [reply]
ty for info 👍

**Comment by aruanlnm** [comment]
hi Can u pls confirm which fast api version is compatible with 
starlette@0.27.0
fastapi==0.103.0 or fastapi==0.101.0 or any other
